channeldb/channel.go

```diff
@@ -3642,6 +3651,7 @@ func putChanInfo(chanBucket kvdb.RwBucket, channel *OpenChannel) error {
 			initialRemoteBalanceType, &remoteBalance,
 		),
 		MakeScidRecord(realScidType, &channel.confirmedScid),
+		tlv.MakePrimitiveRecord(channelMemoType, &channel.Memo),
```

Comment:
Even though I like small commits that are easy to review, this PR is a bit on the "too small" side IMO. So things that are very closely related (as for example the encoding and decoding of the same record) should be grouped together, to make it easier to make sure nothing is forgotten. Especially if it's just a couple of lines and not hundreds of them. So I would combine this and the next commit.